tests/test_utils.go

```diff
@@ -335,7 +336,31 @@ func removeURLSecrets(u *url.URL) string {
 
 // MatchInteraction checks if the request matches a store request in the given cassette.
 func MatchInteraction(r *http.Request, i cassette.Request) bool {
-	return r.Method == i.Method && removeURLSecrets(r.URL) == i.URL
+	// Default matching on method and URL without secrets
+	if !(r.Method == i.Method && removeURLSecrets(r.URL) == i.URL) {
+		return false
+	}
+
+	if r.Body == nil {
+		return true
+	}
+
+	var b bytes.Buffer
+	if _, err := b.ReadFrom(r.Body); err != nil {
+		return false
+	}
+	r.Body = ioutil.NopCloser(&b)
+
+	// Body must be empty or equal
+	if strings.HasPrefix(r.Header["Content-Type"][0], "multipart/form-data") {
+		fmt.Printf("skip %s ??? %s", b.String(), i.Body)
+		return true
+	}
```

Comment:
What does this mean ?

Reply:
This was a hot fix for `multipart/form-data` that are not exactly the same for some reason even when the content is the same.